docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_title: BackgroundTask
+title: Background Task
+description: A library that provides an API for running background tasks.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-background-task'
+packageName: 'expo-background-task'
+platforms: ['android', 'ios']
+isNew: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { PlatformTags } from '~/ui/components/Tag';
+
+`expo-background-task` provides an API to run deferrable background tasks in a way that optimises battery and power consumption for the end user. This module uses the [`BGTaskScheduler`](https://developer.apple.com/documentation/backgroundtasks/bgtaskscheduler) API on iOS and the [`WorkManager`](https://developer.android.com/topic/libraries/architecture/workmanager) API on Android to schedule tasks. It also uses the [expo-task-manager](task-manager.mdx) Native API for running JavaScript tasks.
+
+## Background tasks
+
+A background task is a deferrable unit of work that is performed in the background, outside the lifecycle of your app. This is useful for tasks that need to be executed when the app is not active, such as syncing data with a server, fetching new content, or even checking if there are any [Expo Updates](updates.mdx).
+
+### When are background tasks run?
+
+The Expo Background Task API leverages each platform to execute tasks at the most optimal time for both the user and the device.
+
+This means that the task may not run immediately after it is scheduled, but it will run at some point in the future if the system decides so.
+
+A background task will only run if the battery has enough charge (or the device is plugged into power) and the network is available. Without these conditions, the task won't execute. The exact behavior will vary depending on the operating system.
+
+### When will they stop?
+
+The native platform APIs will stop running background tasks if the user explicitly kills the app (they will be resumed when the user restarts the app) or if the app is uninstalled. If the device is rebooted or the app crashes, the background tasks will be resumed.
```

Comment:
just wanted to double check - is this accurate? there isn't any way to run a task when the app has been killed?

Reply:
As discussed in our sync discussion I'll add a bit more info about how this works on Android.